examples/manual/main.fmf

```diff
@@ -1 +1,10 @@
-test: manual
+summary: Basic manual test
+description:
+    Basic manual test example demonstrates using MarkDown with defined
+    structure to describe actions necessary for its execution.
+    Adhering to its structure allows export to other formats or services.
+    Note any sections may contain shell script code which
+    could be used for semi-automated execution in the future.
+
+test: manual.md
+manual: true
```

Comment:
if we agree on markdown and .md, do we need this ?

Reply:
As discussed on the meeting: We do - E.g. filtering (out) all manual cases